src/components/NavBar/NavIcon.tsx

```diff
@@ -6,10 +6,11 @@ import * as styles from './NavIcon.css'
 interface NavIconProps {
   children: ReactNode
   isActive?: boolean
+  label?: string
   onClick: () => void
 }
 
-export const NavIcon = ({ children, isActive, onClick }: NavIconProps) => {
+export const NavIcon = ({ children, isActive, label = 'navigation button', onClick }: NavIconProps) => {
```

Comment:
These should be sentence-cased and translated:

    import { t } from '@lingui/macro'
    ...
    export const NavIcon = ({ children, isActive, label = t`Navigation button`, onClick }: NavIconProps) => {